### ISSUE TITLE
Improve the descriptions under “Collections”

### ISSUE THREAD

**Issue by chasenlehara** [canjs/canjs#2871]
Each of the descriptions under the “Collections” section of the home page needs to be very clear; for example, `can-set` “Service modeling” isn’t really informative.

**Comment by gKreator** [comment]
I agree. I think the docs are fairly light at the moment since they just launched.

**Comment by frank-dspeed** [comment]
i think you should let it this way and we create some kind of wiki lexicon that simply has more details about this defined sentences like "Service modeling" gets a extra page or block where we explain what is service modling
